src/coreclr/jit/codegenarmarch.cpp

```diff
@@ -1169,14 +1170,19 @@ void CodeGen::genPutArgSplit(GenTreePutArgSplit* treeNode)
 
             if (regIndex >= treeNode->gtNumRegs)
             {
+                if (firstOnStackOffs == UINT_MAX)
+                    firstOnStackOffs = use.GetOffset();
+
```

Comment:
Surprised the formatter didn't flag this:

```suggestion
                if (firstOnStackOffs == UINT_MAX)
                {
                    firstOnStackOffs = use.GetOffset();
                }
```